README.md

```diff
@@ -92,6 +104,10 @@ Snippet 7 - Big O:
 end
 ```
 
+The above is O(n^2) since we have a nested loop that might need to go over all the (unsorted) elements again.
+When I tried this code, I noticed it wouldn't run, and I believe the reason is an index error. The range should start at 1, and the while loop condition should be i >= 0.
```

Comment:
Yeah -- someone fixed it for me. I stole that code from the internet :) It's because the .each should have been exclusive instead of inclusive. I think.